Packs/Core/Playbooks/playbook-Uncommon_remote_scheduled_task_created.yml

```diff
@@ -0,0 +1,1433 @@
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1070,
+          "y": 1640
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: b6d11f6e-a28a-459a-8004-bec570e4b02a
+    type: title
+    task:
+      id: b6d11f6e-a28a-459a-8004-bec570e4b02a
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "68"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": -130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 6d96992e-fe69-4b71-8e3c-9f64ce6a2aec
+    type: title
+    task:
+      id: 6d96992e-fe69-4b71-8e3c-9f64ce6a2aec
+      version: -1
+      name: Investigation on remote host
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "31"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: 58967e13-7736-4385-858d-85a8966dacd3
+    type: regular
+    task:
+      id: 58967e13-7736-4385-858d-85a8966dacd3
+      version: -1
+      name: Search for related alerts on the remote host
+      description: This task searches for XSIAM agent related alerts on the remote endpoint from the past 24 hours, if an agent is installed.
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "32"
+    scriptarguments:
+      fromdate:
+        simple: 1 day ago
+      ignore-outputs:
+        simple: "false"
+      query:
+        simple: agent_ip_addresses:${Core.OriginalAlert.event.actor_remote_ip}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 1145
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 789cf6e0-eded-4b32-8108-8091409a2537
+    type: condition
+    task:
+      id: 789cf6e0-eded-4b32-8108-8091409a2537
+      version: -1
+      name: Found any alerts indicating this is a malicious scheduled task?
```

Comment:
I think this should be renamed. as you search fir malware category alerts, not specific to a schedule task

Reply:
changed to: "Found any alerts of malicious activity on the remote host?"